README.md

```diff
@@ -1 +1,97 @@
-# news-explorer-frontend
+# Product subject:
+
+## news-explorer-frontend
+
+
+# project description:
+  the frontend part of my final project
+
+
+# Technologies and techniques:
+*  FIGMA
+*  BEM method
+*  HTML
+*  CSS
+*  Responsive Layout
+*  Flexbox
+*  Grid
+*  Media queries
+*  Java Script
+*  Forms
+*  OOP Developing
+*  Webpack
+*  Bable
+*  Form validation
+*  React
+*  API requests
+*  Node.js 
+*  Postman
+*  Express
+*  MongoDB
+
+
+# Project Link:
+  # https://github.com/nitsnats/news-explorer-frontend.git
```

Comment:
* You need to **deploy** the app
* In ` README.md` you need to add some info about your project: technologies, how to download, how to start, how to deploy the app. And you need to add the link to your **deployed app**.